src/v2/Apps/Auctions/Components/AuctionArtworksRailPlaceholder.tsx

```diff
@@ -7,33 +7,37 @@ interface FairExhibitorRailPlaceholderProps {
   done?: boolean
 }
 
+const AUCTION_ARTWORKS_RAIL_HEIGHT = 233
```

Comment:
**suggestion [non-blocking]:** extract hardcoded heights to exported object

This knowledge is spreading, maybe defining a const of heights would be helpful:

```ts
// maybe in a helpers file?
export const auctionHeights = {
  artworksRail: 233,
  artworksImage: 160
}

// then later:
import { auctionHeights } from "./some/helpers/file.ts"
<Box height={auctionHeights.artworksRail} />
```

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
Updated per suggestion. I extracted the values out to a Helper file and updated all usages of it.

Reply:
Much better with the shared constant! I made one lil fix in my pr tho -- we already had a `Utils` directory in the app folder so moved this there, from Helpers (we generally use `Utils` elsewhere).